source/SpinalHDL/Data types/bits.rst

```diff
@@ -276,7 +276,7 @@ Misc
      - Return the range (0 to x.high)
      - Range
    * - x.valueRange
-     - Return the range (x.minValue downto x.maxValue). Note can't be used for value which overflow the JVM Int capacity.
+     - Return the range of valid x values, interpreted as an unsigned integer (0 to 2**width - 1).
```

Comment:
Won't `**` be interpreted by rst as bold?

The formulation can be taken from you earlier modifications:

```rst
- Return the range from minimum to maximum possible value of x (0 to 2 ** width - 1).
```

Reply:
done, I checked and `**` renders fine since it's never closed it seems, I still escaped it